include/datatypes.h

```diff
@@ -221,4 +212,46 @@ typedef enum {
 
 #pragma pack(pop)
 
+static uint32_t ehsm_get_rsa_padding_size(uint32_t padding_mode)
+{
+    switch (padding_mode) {
+        case RSA_PKCS1_OAEP_PADDING:
+            return RSA_PKCS1_OAEP_PADDING_SIZE;
+        case RSA_PKCS1_PADDING:
+            return RSA_PKCS1_PADDING_SIZE;
+        default:
+            return 0;
+    }
+}
+
+static uint32_t ehsm_get_rsa_max_encryption_size(const uint32_t keyspec, const uint32_t padding_mode)
```

Comment:
It's used in other files, why define it as static?
And same @line 241

Reply:
statis function to avoid multiple definition